### PR TITLE
Updated the release notes with visualizer changes.

### PR DESCRIPTION
**Pull request by moorepants**
Fixes issue #89.
Also fixes some spelling errors.

- [x] There are no merge conflicts.
- [x] If there is a related issue, a reference to that issue is in the
  commit message.
- [x] The bug fix is documented in the [Release
  Notes](https://github.com/pydy/pydy#release-notes).
- [x] All reviewer comments have been addressed.

### CODE REVIEW

**Comment by moorepants** [comment]
@pydy/pydy-developers This needs review and merging. Simple README changes.

File: README.rst [hunk 1]
```diff
@@ -343,11 +343,29 @@ User Facing
...
+  - The visualizer can now be loaded in an IPython notebook via IPython's
+    widgets.
+  - A slider was added to manually control the frame playback.
+  - The visualization objects can be manipulated via the GUI.
+  - The scene json file can be edited via the GUI.
+  - The data and scene description have been separated into two json files.
+  - Different simulation files can be loaded via the GUI.
+  - Shapes can now have a material.
+  - Simulation parameters can be modified and the simulations can be rerun all
+    via the GUI.
```
Comment (jcrist) [inline README.rst]: Does all of this actually work now? If not, we should probably wait to add it to the release notes.

Comment (moorepants) [inline README.rst]: Yes, it essentially works. But even so, all of these changes are merged into master already, so the release notes should reflect what is in master.

I'm starting to work on the viz bugs associated with these changes now. But I don't see any reason not to merge this pr.

Comment (jcrist) [inline README.rst]: As long as these are actually implemented and working when we release, then yeah, everything's fine. I'd just hate to say "The visualizer can now be loaded in an IPython notebook via IPython's widgets", and have that not actually be true when we release. Not a -1, just a note. Looks good.

Comment (moorepants) [inline README.rst]: Sure, if any of these things are not true come time for a release it needs to be modified. Right now ipython notebooks work except that specifying cameras no longer works.

**Comment by moorepants** [comment]
If you think this is good, check off the checklist and merge.

**Pull request merged**